mep/books/templates/books/work_list.html

```diff
@@ -16,8 +16,19 @@
 {% include 'snippets/breadcrumbs.html' %}
 <section class="books search-form">
     <form id="books-form">
+        {% render_field form.query %}
+        <label for="{{ form.query.id_for_label }}"></label>
         {% include 'common/form_errors.html' %}
-        <input type="submit" value="submit"/>
+        <fieldset class="group">
+            <legend>Filter by:</legend>
+            <div class="inner">
+                <div class="creators tab" role="tab" aria-setsize="2" aria-posinset="1" aria-selected="false" aria-controls="creators-panel" aria-disabled>Creators*</div>
+                <div class="publishers tab" role="tab" aria-setsize="2" aria-posinset="2" aria-selected="false" aria-controls="publishers-panel" aria-disabled>Publishers*</div>
+                <div class="tabpanel" id="creators-panel" role="tabpanel"></div>
+                <div class="tabpanel" id="publishers-panel" role="tabpanel"></div>
+            </div>
+            <input type="submit" value="submit"/>
+        </fieldset>
```

Comment:
Please remove (or comment out, if you prefer). The only filter we're building in the short term is borrowing dates.